internal/mockable/mockable.go

```diff
@@ -33,6 +33,7 @@ type Session struct {
 	MockableResolverIP           string
 	MockableSoftwareName         string
 	MockableSoftwareVersion      string
+	MockableSubmitter            model.Submitter
```

Comment:
TODO(@bassosimone): we should either remove this part of the diff or finish upgrading the mocks.